util/delete_entities.py

```diff
@@ -27,10 +28,17 @@ def parse_args():
     return parser.parse_args()
 
 
-def delete_entities(entity_type, entities):
-    fapi.delete_entity_type(args.workspace_namespace, args.workspace_name,
-                            entity_type, entities)
-    fapi._check_response_code(resp, 200)
+def _chunk_list(l, size):
+    return [
+        l[i:i + size] for i in xrange(0, len(l), size)
+    ]
+
+
+def delete_entities(args, entity_type, entities):
+    for chunked in _chunk_list(entities, ENTITY_CHUNK_SIZE):
```

Comment:
I find something like this easier to read:

for i from 0 to len(entities)/ENTITY_CHUNK_SIZE+1:
    chunk = entities[i*chunk:i*chunk+ENTITY_CHUNK_SIZE]
    resp = fapi.delete_entity_type(args.workspace_namespace, args.workspace_name,entity_type, chunk)

Reply:
Done.